pages/learn/deploy/deploy-with-balena-button.md

```diff
@@ -87,6 +87,19 @@ version: 3.5.2
   - `applicationConfigVariables` - [Configuration variables][configuration] allow you to provide runtime configuration to the host OS and supervisor. These variables all begin with `BALENA_` or `RESIN_`.
   - `defaultDeviceType` - The device type that will be pre-selected in the "Create fleet" modal. It defaults to Raspberry Pi 4 if not provided. You can find a list of [device types](/reference/hardware/devices/) here.
   - `supportedDeviceType` - The device types that the fleet supports. You can find a list of [device types](/reference/hardware/devices/) here.
+- `version` - A user-defined release version value in the 3-digit format "1.2.3" that is
+  shown in the Version column of the Releases page of the balenaCloud web dashboard, and
+  which may also be queried through the balena API `release.semver` field. Currently only
+  the 3 core _major.minor.patch_ semver digits may be defined by the user (i.e. no support
+  for user-defined pre-release or metadata [semver syntax](https://semver.org/)). We plan
+  to support the full semver syntax in user-defined version values in the future. If the
+  version value follows the 3-digit format "1.2.3" and multiple deployments are made using
+  the same version value, the balena API will automatically increase the revision value in
+  the format "1.2.3+rev1", "1.2.3+rev2", "1.2.3+rev3" and so on. If the version value does
+  **not** follow the 3-digit format, currently deployment may still succeed as long as the
+  version value is unique across all releases in the fleet, however the value will not
+  surface in the web dashboard (which will then typically display version `0.0.0`) and
+  further deployments may not succeed in the future as non-conformant usage.
```

Comment:
How about separating the paragraph with a line break for the deprecated part and simplify it a bit:
```suggestion
  the format "1.2.3+rev1", "1.2.3+rev2", "1.2.3+rev3" and so on.  
  Using version values that do **not** follow the 3-digit format is deprecated
  and may be blocked in the future. Even though currently such deployments
  may still succeed as long as the version value is unique across all
  releases in the fleet, the web dashboard will display version `0.0.0`
```

Reply:
I wasn't convinced about the line break (because the description appears in indented bullet points), but I incorporated the wording simplification 👍  with minor changes.